batcher/aligned-batcher/src/lib.rs

```diff
@@ -1226,6 +1231,13 @@ impl Batcher {
                 ))?;
         }
 
+        // Update metrics
+        let queue_len = batch_state_lock.batch_queue.len();
+        let queue_size_bytes = calculate_batch_size(&batch_state_lock.batch_queue)?;
+
+        self.metrics.queue_len.set(queue_len as i64);
+        self.metrics.queue_size_bytes.set(queue_size_bytes as i64);
```

Comment:
I think it makes sense to this logic to `aligned-batcher/metrics.rs` and do:
```suggestion
     self.metrics.update_queue_metrics(queue_len, queue_size_bytes);
```

Mostly because they are *"measuring the same"* and we always want to update them at the same time.

Reply:
Yes, I think it's a good idea 👌🏻